apps/pipelines/models.py

```diff
@@ -111,10 +111,60 @@ def create_default(cls, team, name=None):
             position={"x": 1000, "y": 200},
             data=FlowNodeData(id=end_id, type=EndNode.__name__, params={"name": "end"}),
         )
-        default_nodes = [start_node.model_dump(), end_node.model_dump()]
+        if llm_provider_id and llm_provider_model:
```

Comment:
A test for this code would be good to ensure that changes in the `LLMResponseWithPrompt` data model don't break this.